doc/dev/development.md

```diff
@@ -286,18 +286,18 @@ passed both unit and integration tests. We only merges pull requests when
   [table driven testing](https://github.com/golang/go/wiki/TableDrivenTests)
 * Concurrent unit test runs must pass.
 
-## Using the mixer
+## Using Mixer
 
-Once you've built the source base, you can run the mixer in a basic mode using:
+Once you've built the source base, you can run Mixer in a basic mode using:
 
-```
+```shell
 bazel-bin/cmd/server/mixs server \
```

Comment:
not blocking, but just as a thought: we will need to update this section very soon.